spine-js/spine.js

```diff
@@ -1443,6 +1450,8 @@ spine.SkinnedMeshAttachment.prototype = {
 					wx += (vx * bone.m00 + vy * bone.m01 + bone.worldX) * weight;
 					wy += (vx * bone.m10 + vy * bone.m11 + bone.worldY) * weight;
 				}
+				slot.vertices[w] = wx - slot.bone.worldX;
+				slot.vertices[w + 1] = slot.bone.worldY - wy;
```

Comment:
`Slot` doesn't have a `vertices` field. Why allocate one? When are the values stored here used?